spec/feature_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'order'
+require 'text'
+require 'menu'
+
+describe 'feature tests' do
+
+    let(:menu) {Menu.new}
+    let(:order) {Order.new}
+
+    it 'allows customers to view the menu' do
+        expect(order.show_menu).to eq menu.view_menu
+    end
+    
+    it 'allows customer to order multiple dishes' do
+        order_hash = {:Coke => 3, :Special_Fried_Rice => 3}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)